error_tracker/django/utils.py

```diff
@@ -182,3 +183,34 @@ def capture_exception(request=None, exception=None, additional_context=None):
     from error_tracker.django.middleware import error_tracker
     error_tracker.capture_exception(request=request, exception=exception,
                                     additional_context=additional_context)
+
+
+def clean_value(x):
+    x = x.value.replace('[["', "").replace('"]]', "").replace('"', "")
+    return x
+
+
+def cookie_parse(headers):
```

Comment:
One last change request, this method does not seem to parse only cookie, it's parsing header. Can we rename this method to `parse_headers`